src/openshift/cluster.ts

```diff
@@ -430,7 +430,7 @@ export class Cluster extends OpenShiftItem {
         try {
             const result = await Progress.execFunctionWithProgress(
                 `Login to the cluster: ${clusterURL}`,
-                () => Cluster.cli.executeTool(Command.odoLoginWithUsernamePassword(clusterURL, username, passwd)));
+                () => Cluster.cli.executeTool(Command.odoLoginWithUsernamePassword(clusterURL, username, passwd), undefined, true));
```

Comment:
The error is thrown correctly, but the incorrect password is in the error popup. I think it would be nice to catch the exception, then rethrow with a modified error message that doesn't contain the password. This would be important in the case that someone is doing a screencast of using the extension and types in their password incorrectly, since in that case, it would expose what the user typed in for the password, which is probably close to the actual password.